#ommp:
  joint_limits:
          back_left_wheel_joint:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: true
                  max_acceleration:  25.0 
                  has_jerk_limits: true
                  max_jerk: 1.0
                  has_effort_limits: false
          back_right_wheel_joint:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: true
                  max_acceleration:  25.0 
                  has_jerk_limits: true
                  max_jerk: 1.0
                  has_effort_limits: false
          camera_dummy_joint:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          front_left_wheel_joint:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: true
                  max_acceleration:  25.0 
                  has_jerk_limits: true
                  max_jerk: 1.0
                  has_effort_limits: false
          front_right_wheel_joint:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: true
                  max_acceleration:  25.0 
                  has_jerk_limits: true
                  max_jerk: 1.0
                  has_effort_limits: false
          joint_0:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_1:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_2:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_3:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_4:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_5:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_finger1:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
          joint_finger2:
                  has_position_limits: false
                  has_velocity_limits: true
                  max_velocity: 2.0
                  has_acceleration_limits: false
                  has_jerk_limits: false
                  has_effort_limits: false
